docs/installing_hypershift_cli.md

```diff
@@ -6,23 +6,24 @@ The Hosted Control Plane command-line interface (CLI) is an `oc` command-line [p
 
 ## Installing the Hosted Control Plane CLI
 
-**NOTE: This document is unofficial will be updated later. For now download the CLI binary from this page
+1. [Enable the Hypershift feature](https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/provision_hosted_cluster_on_mce_local_cluster.md#configuring-the-hosting-service-cluster)
 
-1. Download the HyperShift CLI binary.
+2. On the OCP console, click on the ? button at the top right hand corner and choose `Command line tools` menu to download the HyperShift CLI binary.
 
+(unofficial: you can download one of these as well)
```

Comment:
Is this going to change to other links at some point to make it official? Otherwise, we should just title it with "Tech Preview" or "Dev Preview"